midi_app_controller/config.py

```diff
@@ -1,17 +1,32 @@
 import os
+import appdirs
+from pathlib import Path
 
 
 class Config:
     """Class that stores configuration."""
 
-    # TODO Find better locations for config files.
-    BINDS_DIRECTORY = os.path.abspath(
-        os.path.join(
-            os.path.dirname(os.path.dirname(__file__)), "config_files", "binds"
-        )
-    )
-    CONTROLLERS_DIRECTORY = os.path.abspath(
-        os.path.join(
-            os.path.dirname(os.path.dirname(__file__)), "config_files", "controllers"
-        )
-    )
+    _CONFIG_DIR = (Path(__file__) / "../../config_files").resolve().absolute()
+    _USER_CONFIG_DIR = (
+        Path(appdirs.user_config_dir(appname="MIDI App Controller")) / "config"
```

Comment:
Is the `/ "config"` needed? I don't see any app in my `.config` directory that does something like that. All of them just store files directly in their root directory.